YDotNet/Infrastructure/ClientIdGenerator.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace YDotNet.Infrastructure;
+
+/// <summary>
+///     Helper class to deal with client ids.
+/// </summary>
+public static class ClientIdGenerator
+{
+    /// <summary>
+    ///     The maximum safe integer from javascript.
+    /// </summary>
+    public const long MaxSafeInteger = 2 ^ (53 - 1);
```

Comment:
@SebastianStehle To get the max. safe value using 53-bit, shouldn't this expression be `(2 ^ 53) - 1`?

I'm asking because, for example, if we were using 3-bit numbers, we'd only be able to represent `0b111` (which is 7). To get 7, we do `(2 ^ 3) - 1` and not `2 ^ (3 - 1)`.

Reply:
Yes, sounds right. I have copied it from somewhere and not thought too much about it.